src/unittest/test_net.c

```diff
@@ -0,0 +1,45 @@
+#include "unittest.h"
+#include "net.h"
```

Comment:
The `#include` directives with quotes are relative to the source file, so this should be `#include "../net.h"`, same for other .h files from the parent directory.